docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -277,6 +283,60 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+## Sending Search Requests from Client to Server
+
+Client application can send search requests which consist of a collection of search conditions.
+Server listens for the incoming search requests and handles them accordingly.
+After handling a request, server sends a response to the client that sent the request.
+
+### Receiving Search Request
+
+To receive and handle search request on the server, follow these steps:
+
+```
+// 1. Get server handle.
```

Comment:
@m-michalski2, I understand your point, however, if we see the other sections, it has been given as a step. It looks readable and easy to understand. 

@mijin85cho, please let us know your views on this.